### ISSUE TITLE
No extension icon 

### ISSUE THREAD

**Issue by kurtextrem** [hharnisc/tabbie#4]
Add this to the manifest:
```
	"icons": {
		"128": "icon.png",
		"16": "icon.png",
		"48": "icon.png"
	},
```

**Comment by NdagiStanley** [comment]
I could look for a catchy icon.

**Comment by hharnisc** [comment]
Good call @kurtextrem! I'll update this in the next release 😄 

Hey @NdagiStanley 👋 up to now I've been using some variation of this icon: https://github.com/hharnisc/tabbie/blob/master/store/icon.png Open to new designs as well if you've got ideas!

**Comment by hharnisc** [comment]
I've got this one fixed for 0.3.0 release - chrome store says it should going out within an hour

**Comment by NdagiStanley** [comment]
Awesome. Got it.
